token-validation-spring-test/src/main/java/no/nav/security/token/support/spring/test/MockLoginController.java

```diff
@@ -60,4 +59,41 @@ public Cookie addCookie(
         }
         return cookie;
     }
+
+    @Unprotected
+    @PostMapping("/cookie")
+    public Cookie addCoaaokie(
+        @RequestParam(value = "issuerId") String issuerId,
```

Comment:
remove and replace with: @PathVariable(value = "issuerId") String issuerId,

Reply:
✅